src/benchmark/tpcc_benchmark.cpp

```diff
@@ -40,13 +40,13 @@ int main(int argc, char* argv[]) {
   // clang-format off
   cli_options.add_options()
     // We use -s instead of -w for consistency with the options of our other TPC-x binaries.
-    ("s,scale", "Scale factor (warehouses)", cxxopts::value<size_t>()->default_value("1")) // NOLINT
-    ("consistency_checks", "Run TPC-C consistency checks after benchmark (included with --verify)", cxxopts::value<bool>()->default_value("false")); // NOLINT
+    ("s,scale", "Scale factor (warehouses)", cxxopts::value<size_t>()->default_value("10"))
```

Comment:
Not really into TPC-DS. What are the effects of increasing warehouses? Simply larger datasets?

Reply:
Larger dataset on the one hand (10 is still small, 1-2 GB maybe), but more importantly the clients all update primarily "their" warehouse. If there is only one, we basically have a rather "unrealistic" scenario and many conflicts. These conflicts often hide other effects.

Most papers work with dozens warehouses, so I think we should do that as well.